### ISSUE TITLE
New device sysinfo that could be sent to puavo 

### ISSUE THREAD

**Issue by asokero** [puavo-org/puavo-os#435]
- [ ] Xrandr video output info
- [ ]  Pulse audio sink info
- [ ]  Images available on the laptop

**Comment by juhaerk** [comment]
Of three, I only implemented one, "Xrandr video output info"... https://github.com/puavo-org/puavo-os/commit/40ae505850bf824036513b914001848a0a9a293e

Pulse audio sink info is a difficult one... the problem is that pulseaudio is running as a separate process for each session user, which means that pulseaudio info may not be available as sysinfo (or is only sporadically), and the relevant command "pacmd" works only when used as part of session that is capable of connecting to pulseaudio.  Even then, the currently manually run command "puavo-pulseaudio-list" is not only passively observing, but changes pulseaudio state, thus even if we could run it in user session, we should not because that would mess up user audio.

Not implementing "Images available on the laptop" either, because that has already been implemented some years ago, not part of sysinfo though.  The ldap-attributes holding that information may be somewhat hidden, though.

As a bonus, though, there is support for battery information: https://github.com/puavo-org/puavo-os/commit/298743213289c77da386e1a8736807db1889a74e and also organisation domain: https://github.com/puavo-org/puavo-os/commit/cec332c83427f5ac3a92d67ef79f9b22d9897b94
